### PR TITLE
[Pro splash pages] New Pricing page for pro

### PR DESCRIPTION
**Pull request by anothertompetty**
This PR adds the initial new content for the updated pricing page. Main feature is the combination of the pricing and comparison tables. 

Note: this is currently only for `pricing.en.js` and effects the UK, Sweden, and Ireland.

A darker menu version will be required, but I'll do the menu in a different PR. FAQ etc. also to be updated at a later date.

image

### CODE REVIEW

**Comment by clippy** [comment]
**I have manually checked the visuals in:**

- [ ] Chrome
- [ ] IE11
- [ ] Firefox
- [ ] Safari
- [ ] iPad

**Comment by anothertompetty** [comment]
@jamesshedden a quick reviewski? :eyes:

File: app/css/components/page-hero.scss [hunk 1]
```diff
@@ -101,6 +101,10 @@
   background: $dark-gray;
 }
 
+.page-hero--one-product-pricing {
+  background: $light-gray;
```
Comment (jamesshedden) [inline app/css/components/page-hero.scss]: Could we just use `.u-background-light-gray` in the template for this? I know there's another class in this file that only adds colour, but not sure how it got there - would be nicer just to re-use utility class in this case I think!

File: app/pages/pricing/pricing.en.js [hunk 2]
```diff
@@ -19,79 +18,100 @@ export default class PricingEn extends React.Component {
     const hasPercentagePricing = getMessage(this.context.messages, 'country_properties.has_percentage_pricing');
     return (
       <Translation locales='en'>
-        <div className='page-hero--pricing page-hero'>
+        <div className='page-hero--one-product-pricing page-hero'>
```
Comment (jamesshedden) [inline app/pages/pricing/pricing.en.js]: @anothertompetty Maybe use background colour util class as mentioned in other comment

File: app/css/components/pricing-options.scss [hunk 3]
```diff
@@ -3,6 +3,10 @@
   width: 752px;
 }
 
+.pricing-options-new {
```
Comment (jamesshedden) [inline app/css/components/pricing-options.scss]: `pricing-options--new` (missing extra dash for modifier)

File: app/pages/pricing/pricing.en.js [hunk 4]
```diff
@@ -19,79 +18,100 @@ export default class PricingEn extends React.Component {
...
           <div className='site-container'>
-            <div className='grid pricing-options u-center u-padding-Bxl'>
+            <div className='grid pricing-options-new u-center u-padding-Bxl'>
```
Comment (jamesshedden) [inline app/pages/pricing/pricing.en.js]: @anothertompetty Mentioned slightly correction to this class name on the CSS file :+1:

**Comment by jamesshedden** [comment]
@anothertompetty Minor comments! Looks good :+1: Did you decided against the version with the greyed out `X`-ed features in the end?

**Comment by anothertompetty** [comment]
Thanks so much @jamesshedden !

**Pull request merged**